From db501ad8fac885647fa073791376c97def0263b3 Mon Sep 17 00:00:00 2001
From: Finley Xiao <finley.xiao@rock-chips.com>
Date: Fri, 21 Jul 2017 15:50:46 +0800
Subject: [PATCH] soc: rockchip: power-domain: vpu and rkvdec ignore idle
 request for rk3366

As vpu and rkvdec use the common idle request, if vpu enable idle request
and rkvdec is working, the rkvdec will be crash, so let them both don't
do idle request.

Change-Id: I1fe2bc99c81d3b2018e86896548514e5e8093a4a
Signed-off-by: Finley Xiao <finley.xiao@rock-chips.com>
---
 drivers/soc/rockchip/pm_domains.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/soc/rockchip/pm_domains.c b/drivers/soc/rockchip/pm_domains.c
index e72f3e0ad1df..7e3853f0f109 100644
--- a/drivers/soc/rockchip/pm_domains.c
+++ b/drivers/soc/rockchip/pm_domains.c
@@ -776,9 +776,9 @@ static const struct rockchip_domain_info rk3366_pm_domains[] = {
 	[RK3366_PD_PERI]	= DOMAIN_RK3368(10, 10, 6, true),
 	[RK3366_PD_VIO]		= DOMAIN_RK3368(14, 14, 8, false),
 	[RK3366_PD_VIDEO]	= DOMAIN_RK3368(13, 13, 7, false),
-	[RK3366_PD_RKVDEC]	= DOMAIN_RK3368(11, 11, 7, false),
+	[RK3366_PD_RKVDEC]	= DOMAIN_RK3368(11, 11, -1, false),
 	[RK3366_PD_WIFIBT]	= DOMAIN_RK3368(8, 8, 9, false),
-	[RK3366_PD_VPU]		= DOMAIN_RK3368(12, 12, 7, false),
+	[RK3366_PD_VPU]		= DOMAIN_RK3368(12, 12, -1, false),
 	[RK3366_PD_GPU]		= DOMAIN_RK3368(15, 15, 2, false),
 };
 
-- 
2.35.3

